### PR TITLE
Add support for iOS 8

### PR DESCRIPTION
**Pull request by joemasilotti**
The only code that the framework uses requiring iOS 9 is a few auto layout APIs. I updated the snapshot layout logic to use iOS 8-compatible auto layout constraints when needed.

One thing to note is that `applicationNameForUserAgent` was added in iOS 9. So if a developer wants to set a custom User Agent for their requests they will need to use a workaround. The following works well for me as long as it is set before the first `WKWebView` is instantiated.

```
NSUserDefaults.standardUserDefaults().registerDefaults(["UserAgent": "CustomAgentString"])
```

I'm happy to add that to the README if you think it would be beneficial.

### CODE REVIEW

**Comment by joemasilotti** [comment]
Friendly ping for this PR.

File: Turbolinks/VisitableView.swift [hunk 1]
```diff
@@ -132,12 +132,21 @@ public class VisitableView: UIView {
             screenshot.translatesAutoresizingMaskIntoConstraints = false
             screenshotContainerView.addSubview(screenshot)
             
-            NSLayoutConstraint.activateConstraints([
-                screenshot.centerXAnchor.constraintEqualToAnchor(screenshotContainerView.centerXAnchor),
-                screenshot.topAnchor.constraintEqualToAnchor(screenshotContainerView.topAnchor),
-                screenshot.widthAnchor.constraintEqualToConstant(screenshot.bounds.size.width),
-                screenshot.heightAnchor.constraintEqualToConstant(screenshot.bounds.size.height)
-            ])
+            if #available(iOS 9.0, *) {
```
Comment (zachwaugh) [inline Turbolinks/VisitableView.swift]: You can ditch this whole `if/else` statement and just use the iOS 8.0 method since the behavior is the same

**Comment by zachwaugh** [comment]
Thanks for working on this!

**Comment by joemasilotti** [comment]
Sure thing, updated!

File: Turbolinks/VisitableView.swift (hunk 1 above)
Comment (joemasilotti) [inline Turbolinks/VisitableView.swift]: Done.

**Comment by sstephenson** [comment]
Thanks Joe!

**Pull request merged**